etc/recipes/lsst-libffi/build.sh

```diff
@@ -0,0 +1,15 @@
+./configure --prefix=$PREFIX || exit 1
+make || exit 1
+make install || exit 1
+
+cd $PREFIX
```

Comment:
Just for the sake of future proofing (if we one day decide our packages should work in directories including spaces), maybe enclose in quotes ("$PREFIX")?

There are a few more of these in other recipes.